components/ProfileButton.vue

```diff
@@ -1,10 +1,24 @@
 <template lang="pug">
 .profile-button(:class="{ 'nuxt-link-active': linkActive }")
   .auth-pending(v-if="isAuthPending"): loading-spinner
+  //- TODO: factor svgs out somehow! make it easier to use and change coclors dynamically
```

Comment:
I think we should be able to use the [`vue-svg-loader`](https://github.com/nuxt-community/svg-module#vue-svg-loader) option of `@nuxtjs/svg` to get the desired behavior.

Reply:
Sounds great.. but I just couldn't get it to work. Looks like once it's loaded it doesn't apple css anymore.. :/

Reply:
Maybe this is caused by scoped styling and the loaded SVG not getting the correct `data-` attribute. I can try to get it working

Reply:
We can change the svgs themselves to use `currentColor`. Still other css has no affect, even when assigning a class to the svg. So to resize it in the dropdown it's still needed to be there.. 
only other thing is that we could extract it into it's own component that just has the raw svg in it's template .-.